elastic/kibana
# [Logs+] Replace usages of link-to routes in infra server

closes [#157985](https://github.com/elastic/kibana/issues/157985)

## 📝  Summary

This PR moves the previously implemented [infra locators](https://github.com/elastic/kibana/pull/155156) to infra common and replaces the last usage of `link-to` routes in the infra public with the appropriate locator.

## ✅  Testing

1. Navigate to Observability Alerts
2. Create a new Log Threshold Rule that should fire
3. Open the alert details flyout
4. Click on the `View in app` button
5. Should correctly navigate to Stream view or Discover in case of serverless.
* All other links to Logs should still work properly as before as well 


https://github.com/elastic/kibana/assets/11225826/9af85ea6-5fdd-4505-9707-6489b760f311
